wgsl/index.bs

```diff
@@ -8710,6 +8710,25 @@ An assignment is a <dfn noexport>full assignment</dfn> if:
 
 Otherwise, an assignment is a <dfn noexport>partial assignment</dfn>.
 
+A <dfn>full reference</dfn> is an expression of [=reference type=] that is one of:
+* an identifier *x* [=resolving|resolves=] to a variable, or
+* `(`*x*`)` where *x* is a [=full reference=], or
+* `*`*p* where *p* is a [=full pointer=].
+
+A <dfn>full pointer</dfn> is an expression of [=pointer type=] that is one of:
+* `&`*r* where *r* is a [=full reference=], or
+* an identifier *p* [=resolving|resolves=] to a [=let-declaration=] initialized to a [=full pointer=], or
+* `(`*x*`)` where *x* is a [=full pointer=], or
+* `(`*p*`)` where *p* is a [=full pointer=].
+
```

Comment:
Suggest adding that a full reference and full pointer encapsulate all the memory locations for *x* and removing from the table below.